bayesian_benchmarks/data.py

```diff
@@ -153,7 +153,7 @@ def read_data(self):
 
 @add_regression
 class Naval(Dataset):
-    N, D, name = 11934, 12, 'naval'
+    N, D, name = 11934, 14, 'naval'
```

Comment:
Is this consistent with line 170?

Reply:
yes the naval dataset has D=16 (and D=14 after two features are removed)

https://archive.ics.uci.edu/ml/datasets/Condition+Based+Maintenance+of+Naval+Propulsion+Plants

Reply:
ah ok great!